src/SFA.DAS.EmployerUsers.Web/Controllers/AccountController.cs

```diff
@@ -154,7 +155,7 @@ public async Task<ActionResult> Logout()
         [HttpGet]
         [Route("account/register")]
         [OutputCache(Duration = 0)]
-        public async Task<ActionResult> Register(string clientId, string returnUrl)
+        public async Task<ActionResult> Register(string clientId, string returnUrl, string firstName, string lastName, string email)

```

Comment:
What's the historical preference on passing user information in the clear in the query string? Could / Should we pass the correlation id along and allow employer users to retrieve the information via an api call the same way registrations does?

Reply:
I don't think there is any historic precedent. I believe the general consensus is that we shouldn't pollute the EmployerUsers solution with MA specific implementation.